gateway/cli/src/lightning_commands.rs

```diff
@@ -90,6 +102,20 @@ impl LightningCommands {
                     .await?;
                 println!("{response}");
             }
+            Self::PayInvoice {
```

Comment:
PayInvoiceSelf

Reply:
Done.

Reply:
Doesn't look updated to me? Did the commit get pushed?